download-data.sh

```diff
@@ -20,9 +20,9 @@ fi
 
 CIFAR_100_DIRECTORY="${DATA_DIRECTORY}/cifar-100-python"
 if [[ ! -d "$CIFAR_100_DIRECTORY" ]]; then
-    curl https://www.cs.toronto.edu/~kriz/cifar-100-python.tar.gz -o "$DATA_DIRECTORY"/cifar-100-batches-py.tar.gz
-    tar -xvf "$DATA_DIRECTORY"/cifar-100-batches-py.tar.gz -C "$DATA_DIRECTORY"
-    rm -rf "$DATA_DIRECTORY"/cifar-100-batches-py.tar.gz
+    curl https://www.cs.toronto.edu/~kriz/cifar-100-python.tar.gz -o $DATA_DIRECTORY/cifar-100-batches-py.tar.gz
+    tar -xvf $DATA_DIRECTORY/cifar-100-batches-py.tar.gz -C $DATA_DIRECTORY
```

Comment:
it is always safer to use the quotes. please put them back in to have it consistent